commons-statistics-descriptive/src/main/java/org/apache/commons/statistics/descriptive/Variance.java

```diff
@@ -124,10 +127,10 @@ public static Variance of(double... values) {
         // To prevent squaredDevSum from spuriously attaining a NaN value
         // when accum2Squared (which implies accum is also infinite) is infinite, assign it
```

Comment:
This comment is now out-of-date.

You should also update the method javadoc with the case: if the sum of the squared deviations from the mean is infinite, the variance is NaN.